deploy/catalog_resources/redhat/1.1.1/businessautomation-operator.1.1.1.clusterserviceversion.yaml

```diff
@@ -18,7 +18,10 @@ metadata:
     operator-businessautomation: "true"
 spec:
   displayName: Business Automation
-  description: Business Automation Operator for deployment and management of RHPAM/RHDM environments.
+  description: |
+    Business Automation Operator for deployment and management of RHPAM/RHDM environments.
+
+    **Look left for a link to the Installer UI.**
```

Comment:
**To use the guided installer to provision an environment, open the Installer link, in the links section on the left side of this page**